DS-Assignment6/MainEmployee.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Assignment 1
+ * Class to realize the employee details
+ * @author Arpit Pittie
+ */
+package session6;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+public class MainEmployee {
+
+	private HashMap<String, Employee> employees;	//Map for mapping the employee id with the details uniquely
+	private Employee details;	//The employee object
+	private Scanner scan;	//to take input from the user
+	private List<String> keys;	//List of keys
+	
+	public MainEmployee() {
+		employees = new HashMap<String, Employee>();
+		keys = new ArrayList<String>();
+		scan = new Scanner(System.in);
+	}
+	
+	/**
+	 * Method to get the details of the employee
+	 */
+	public void getDetails() {

```

Comment:
This should return the employ list to match the name of the method i.e getDetails. If in some case you don't to return you can name method as loadDetails or populated details. Get is something that return some value.